docs/src/pages/premium-themes/paperbase/Header.tsx

```diff
@@ -41,7 +41,7 @@ export default function Header(props: HeaderProps) {
             <Grid item xs />
             <Grid item>
               <Link
-                href="#"
+                href="https://docs.paperbase.app/"
```

Comment:
This is not the documentation of the template https://deploy-preview-27063--material-ui.netlify.app/premium-themes/paperbase/. Could it confuse developers?